src/Microsoft.VisualStudio.ProjectSystem.FSharp.VS/ProjectSystem/VS/FSharpProjectSelector.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Xml.Linq;
+using System.Xml.XPath;
+using System.Runtime.InteropServices;
+using System.Xml;
+using Microsoft.VisualStudio.Packaging;
+using Microsoft.VisualStudio.ProjectSystem.VS.Generators;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    [ComVisible(true)]
+    [Guid(SelectorGuid)]
+    [ClassRegistration(SelectorGuid, AssemblyQualifiedSelectorTypeName)]
```

Comment:
The ClassRegistrationAttribute lives in the base dll and Registration requires the assembly qualified name to find the type in a dll different from the one that the attribute lives in. Needed to tweak the ClassRegistrationAttribute a bit for this instead of duplicating the attribute in this dll.